### ISSUE TITLE
 Is it possible to increase censorship of women’s legs?

### ISSUE THREAD

**Issue by sirius1wbe** [silveredgold/beta-protection#81]
I want to know if this AI model can recognize female legs, or thighs and calves and process them, it would be very practical, sometimes I want a harsher censor

**Comment by alsidgjls** [comment]
Yes, you can make a config file using json or yml that gives you access to the harshness of censoring of all or some things that are recognized by the AI as well as some other useful settings. The page that teaches you about these settings is kind of hard to find, so much so that I'm having trouble finding it myself, but I know it's there, and you can probably find it by snooping around the beta protection and beta censoring pages.
